packages/ui/src/components/va-stepper/VaStepperStepButton.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <li
+    class="va-stepper__step-button"
+    :class="{
+      'va-stepper__step-button--active': $props.modelValue >= $props.stepIndex,
+      'va-stepper__step-button--disabled': step.disabled || isNextStepDisabled($props.stepIndex),
+      'va-stepper__step-button--navigation-disabled': $props.navigationDisabled,
+    }"
```

Comment:
Let's rewrite this with a `useBem` composable. [Example](https://github.com/epicmaxco/vuestic-ui/blob/develop/packages/ui/src/components/va-badge/VaBadge.vue#L56).